WordPress/src/test/java/org/wordpress/android/ui/sitecreation/domains/SiteCreationDomainsViewModelTest.kt

```diff
@@ -76,9 +87,13 @@ class SiteCreationDomainsViewModelTest {
 
     private fun <T> testWithSuccessResponse(
         queryResultSizePair: Pair<String, Int> = MULTI_RESULT_DOMAIN_FETCH_QUERY,
+        domainValidResultType: ValidationResultType = VALID_AVAILABLE,
         block: suspend CoroutineScope.() -> T
     ) {
         test {
+            whenever(siteCreationDomainValidator.validateDomain(any())).thenReturn(
```

Comment:
Here, I add the domain validation logic since in the `ViewModel` the check is being run there once the domains have been successfully fetched.